src/services/mongoose/users-ips.ts

```diff
@@ -57,6 +57,6 @@ export const UsersIpRepository = (): IUsersIPsRepository => {
 const userIPsFromRaw = (result: UserIPsType): UserIPs => {
   return {
     id: result.id as UserId,
-    lastIPs: (result.lastIPs || []) as IPType[],
+    lastIPs: result.lastIPs as IPType[],
```

Comment:
This cast breaks the `lastIps: IPType` contract defined in `UserIPs` type since `result.lastIPs` can also be `undefined`

Reply:
I've added undefined to the `UserIPS` type.

Reply:
I don't think it should every be undefined? https://github.com/GaloyMoney/galoy/blob/5ee38ade1ab2c7b87cb6b3c66e78a03f6abc0b07/src/services/mongoose/schema.ts#L152

Reply:
You're right it's the `ip` in the `IpType` which could be undefined - not the `IpType` itself.